index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Static Web Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<h1> Static Web Quiz</h1>
+
+  <article class="article">
+    <h1 class="cardTitle"> Red Cards </h1>
+      
+      <section> 
+        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Red_card.svg/220px-Red_card.svg.png" class="cards">
```

Comment:
`<img>` tags are sort of the platypus of elements. They take on properties of both inline and block tags. To make sure you have a block level element try wrapping your image in a `<div>` tag.